File: activity-1/components/Container.js
export class Container {
	/**
	 *
	 * @param {number} ind - The number of containers already present
	 */
	constructor(ind, containerModel) {
		const body = document.querySelector("body");

		const newContainer = document.createElement("canvas");
		newContainer.id = `container${ind + 1}`;
		newContainer.classList.add("container");

		containerModel.updateContainersList(newContainer);

		containerModel.containerCount++;

		body.appendChild(newContainer);
	}
}

File: ScriptSharpProject 1/ScriptSharpProject/ScriptSharpProjectTemplate/components/Container.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Html;

namespace Kepler.ScriptSharpProjectTemplate.components
{
    class Container
    {
        Container(int ind, Object containerModel)
        {
            Element body = Document.QuerySelector("body");

            Element newContainer = Document.CreateElement("canvas");

            newContainer.ID = "container" + ind + 1;

            newContainer.ClassName = "container";

            containerModel.updateContainersList(newContainer);

            containerModel.containerCount++;

            body.appendChild(newContainer);
        }
    }
}
